src/core/client/admin/routes/Moderate/SiteSelector/SiteSelector.tsx

```diff
@@ -9,6 +9,7 @@ import { AppWindowIcon } from "coral-ui/components/icons";
 import SiteSelectorCurrentSiteQuery from "./SiteSelectorCurrentSiteQuery";
 import SiteSelectorSite from "./SiteSelectorSite";
 
+import { Divider } from "coral-ui/components/v2";
```

Comment:
Can you please move this to the correct import grouping?

Reply:
✅